src/super_gradients/training/transforms/all_transforms.py

```diff
@@ -118,3 +124,25 @@
     Transforms.RandomAutocontrast: RandomAutocontrast,
     Transforms.RandomEqualize: RandomEqualize,
 }
+logger = get_logger(__name__)
+
+try:
+    from albumentations import BasicTransform
+
+    imported_albumentations_failure = None
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    logger.debug("Failed to import pytorch_quantization")
+    imported_albumentations_failure = import_err
+
+if imported_albumentations_failure is None:
+    ALBUMENTATIONS_TRANSFORMS = {
+        name: cls for name, cls in inspect.getmembers(importlib.import_module("albumentations"), inspect.isclass) if issubclass(cls, BasicTransform)
+    }
+    ALBUMENTATIONS_COMP_TRANSFORMS = {
+        name: cls
+        for name, cls in inspect.getmembers(importlib.import_module("albumentations.core.composition"), inspect.isclass)
+        if issubclass(cls, BasicTransform)
```

Comment:
BasicCompose instead of BasicTransform here?

Reply:
fixed in 77563f1e7432552e70ce693d1003454f0af515d9